app/src/main/java/co/copperhead/pdfviewer/PdfViewer.java

```diff
@@ -40,13 +61,14 @@
 
     private WebView mWebView;
     private Uri mUri;
-    int mPage;
-    int mNumPages;
+    public int mPage;
```

Comment:
Yeah, that probably makes sense. I missed that when looking at this in isolation.